net-libs/ixwebsocket/ixwebsocket-11.4.6.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Websocket and http client and server library with TLS support"
+DESCRIPTION="IXWebSocket is a C++ library for WebSocket client and server development"
+HOMEPAGE="https://github.com/machinezone/IXWebSocket"
+COPYRIGHT="2018-2025 Machine Zone, inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/machinezone/IXWebSocket/archive/refs/tags/v11.4.6.tar.gz"
+SOURCE_DIR="IXWebSocket-$portVersion"
+CHECKSUM_SHA256="c024334f8e45980836c67008979a884d6dcc5ef067dd2eb1fa7241f4c17ddc32"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ixwebsocket$secondaryArchSuffix = $portVersion
+	lib:libixwebsocket$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	ixwebsocket${secondaryArchSuffix}_devel = $portVersion
+	devel:libixwebsocket$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES_devel="
+	ixwebsocket$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	cmake -Bbuild \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DBUILD_SHARED_LIBS=true \
+		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
+		-DCMAKE_INSTALL_LIBDIR=$libDir \
+		-DUSE_TLS=true
+	make -C build $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
```

Comment:
Needs additional changes to get files into the devel package (eg prepareInstalledDevelLib/packageEntries ...)